Dockerfile

```diff
@@ -20,5 +20,6 @@ RUN rustc -V; cargo -V; rustup -V; cargo test --all && cargo build --release
 
 FROM debian:buster-slim
 COPY --from=builder /zebra/target/release/zebrad /
+RUN echo "[tracing]\nendpoint_addr = '0.0.0.0:3000'" > /zebrad.toml
 EXPOSE 3000 8233 18233
```

Comment:
Hmm, doesn't this mean that we're exposing the internal RPC port to the internet?

Reply:
The firewall rules for the tracing port (3000) only allow IPs from known gcloud health checkers to talk to it:

![image](https://user-images.githubusercontent.com/552961/90179864-1e2a6700-dd7c-11ea-88d9-18420794f9be.png)
https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/cd.yml#L59

Our rule for the zcash ports (8233, 18233) are open to the entire internet:
![image](https://user-images.githubusercontent.com/552961/90180108-7f523a80-dd7c-11ea-98cc-ee56e3c07d76.png)

https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/cd.yml#L67

Reply:
Awesome, thanks!